### PR TITLE
Nix & Sourcehut docs

### PR DESCRIPTION
**Pull request by toastal**

### CODE REVIEW

File: site/index.html [hunk 1]
```diff
@@ -167,6 +167,7 @@ <h2 id="downloads">Download and install</h2>
       I&apos;ll say that it&apos;s linked statically with <a href="https://www.musl-libc.org/">musl</a>, so it will work on any Linux-based OS,
       not just GNU/Linux.</span></dt>
   <dd><soupault-release platform="linux-x86_64" />
+  <dd>Nix: <a href="https://search.nixos.org/packages?channel=unstable&show=soupault&type=packages&query=soupault">nixpkgs</a></dd>
```
Comment (dmbaturin) [inline site/index.html:170]: The only thing I don't like is how that link is going to be in the middle of the links to binary builds.

Maybe add two separate sections under "Download and install": "Official builds" and "Distro packages"?

**Review by dmbaturin — CHANGES_REQUESTED** [review]

File: site/index.html (hunk 1 above)
Comment (toastal) [reply site/index.html]: This better?

**Comment by dmbaturin** [comment]
Thanks! I may edit the layout a bit, later.

**Comment by toastal** [comment]
Sounds good to me.
-- 
toastal ไข่ดาว | https://toast.al
7944 74b7 d236 dab9 c9ef. e7f9 5cce 6f14 66d4 7c9e

**Pull request merged**